### ISSUE TITLE
Search UI perfomance issues

### ISSUE THREAD

**Issue by gbtb** [ponylang/ponylang-website#488]
Hello. 
I'm experiencing huge slowdown during first opening of search dropdown after a documentation page (re)load. I click inside search text edit, dropdown starts to open and then it hangs for about 4 to 8 seconds not reacting to keystrokes. After it unfreezes, buffered keystrokes gets displayed, but still its quite annoying, since as a newcomer I tend to open many tabs with docs and travel between them a lot.

I've suspected a slow network query, so I've looked into developer tools and looks like its not a query but some slow promise.
![screen-2019-09-29-14:05:51](https://user-images.githubusercontent.com/37017396/65826237-930bc400-e2c4-11e9-9ba6-6c2bbad80de2.png)
I'm using Firefox 69.0.1 (64-bit) with Arch Linux on an old but decent notebook with intel i5 and 8gb of ram.

**Comment by SeanTAllen** [comment]
What search are you referring to? Can you provide a link to an exemplar URL? The site that this repo is for has no search. I suspect you mean https://stdlib.ponylang.io/. If that is the case, the repo for that would be https://github.com/ponylang/mkdocs-theme.

**Comment by gbtb** [comment]
You are right, I was talking about search on https://stdlib.ponylang.io/ . Thanks for re-direction. I will open issue there.

**Comment by SeanTAllen** [comment]
@gtb no need to open an issue there! I can transfer this one.
